### PR TITLE
Scalability, Abailability, Alcohol

### PR DESCRIPTION
**Pull request by s-tammy-kato**
Add some words :)

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -188,6 +188,9 @@ Virtual machine | 仮想マシン | かそうマシン |
 Server layout, architecture | サーバ構成 | サーバこうせい |
 Static IP | 固定IP | こていIP |
 Load | 負荷 | ふか |
+Scalability | 拡張性 | かくちょうせい |
+Abailability | 可用性 | かようせい |
+Alcohol | お酒 | おさけ |
```
Comment (ronkorving) [inline ReadMe.md]: We try to limit this list to IT words only. I like alcohol, but maybe not in this list. Could you remove it?

File: ReadMe.md [hunk 2]
```diff
@@ -188,6 +188,9 @@ Virtual machine | 仮想マシン | かそうマシン |
 Server layout, architecture | サーバ構成 | サーバこうせい |
 Static IP | 固定IP | こていIP |
 Load | 負荷 | ふか |
+Scalability | 拡張性 | かくちょうせい |
+Abailability | 可用性 | かようせい |
```
Comment (ronkorving) [inline ReadMe.md]: The correct English would be "Availability". Could you fix that?

**Comment by ronkorving** [comment]
Thank you for your contribution! I left a few comments, could you please have a look?

File: ReadMe.md (hunk 1 above)
Comment (s-tammy-kato) [reply ReadMe.md]: Sorry, I will remove "alcohol".

File: ReadMe.md (hunk 2 above)
Comment (s-tammy-kato) [reply ReadMe.md]: Oops! Thanks!

**Comment by ronkorving** [comment]
Thank you very much! I will merge it now!

**Pull request merged**